elasticapm/traces.py

```diff
@@ -48,8 +48,8 @@ def __init__(self, tracer, transaction_type="custom", trace_parent=None, is_samp
     def end_transaction(self):
         self.duration = _time_func() - self.start_time
 
-    def begin_span(self, name, span_type, context=None, leaf=False, tags=None):
-        parent_span = get_span()
+    def begin_span(self, name, span_type, context=None, leaf=False, tags=None, parent_span_id=None):
```

Comment:
is this part of the agent API? if so, what do you think about making this private and keep a public `begin_span` without the `parent_span_id` argument?

Reply:
Did you think of something like this? d33852df89124